packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -1072,7 +1073,7 @@ export default mixins(
 }
 
 .expression {
-	textarea, input {
+	textarea:not([disabled]), input:not([disabled]) {
```

Comment:
hmmm... we still want expressions inputs to be clickable

Reply:
even when disabled... to make it clear to users they can click that to open Expressions editor modal

Reply:
Oh, yeah I just check how it works on older versions. Was wondering why is cursor forced there. Will revert back.